packages/cli-kit/src/public/node/api/app-management.ts

```diff
@@ -60,6 +62,7 @@ export async function appManagementRequestDoc<TResult, TVariables extends Variab
   token: string,
   variables?: TVariables,
   cacheOptions?: CacheOptions,
+  preferredBehaviour?: RequestModeInput,
```

Comment:
I don't love that we already have a few optional positional parameters here, let's not add more. Also I don't know if `preferredBehavior` is as descriptive as I'd want. Preferred behavior regarding what exactly?

Reply:
Agree with the naming 👌 
I'm ok with using 
```
requestOptions?: {
    requestMode?: RequestModeInput,
  }
```
too, but maybe we should refactor `appManagementRequestDoc` to receive a single object instead of positional arguments (that can be done in a different PR to not block this)